packages/datagrid/src/components/DefaultHeaderRenderer/DefaultHeaderRenderer.component.js

```diff
@@ -38,7 +37,7 @@ export default function DefaultHeaderRenderer({ column, displayName, onFocusedCo
 				</div>
 			</button>
 			{column.colDef[TALEND_QUALITY_KEY] && (
-				<QualityBar
+				<TranslatedQualityBar
```

Comment:
No need to call it `Translated` if it's translated by default.

Reply:
i can't call QualityBar because i can't use the same name of the named export.